apps/web/modules/ui/components/dropdown-menu/index.test.tsx

```
@@ -0,0 +1,322 @@
+  test("renders dropdown menu with shortcut with dangerouslySetInnerHTML", async () => {
+    const user = userEvent.setup();
+
+    render(
+      <DropdownMenu>
+        <DropdownMenuTrigger data-testid="trigger">Menu</DropdownMenuTrigger>
+        <DropdownMenuContent>
+          <DropdownMenuItem>
+            Item
+            <DropdownMenuShortcut
+              data-testid="menu-shortcut-html"
+              dangerouslySetInnerHTML={{ __html: "&#8984;K" }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)